mainnet/docker-compose.yml

```diff
@@ -2,80 +2,80 @@ version: '3'
 
 services:
  observer-0:
-   image: elrondnetwork/${NODE_TAG}
+   image: ${ORGANIZATION}/${OBSERVER_IMAGE}
    ports:
      - "8080:8080"
      - "10000:37373"
    environment:
      - SHARD=0
    volumes:
-     - ~/MyObservingSquad/node-0/db:/go/elrond-go/cmd/node/db
-     - ~/MyObservingSquad/node-0/logs:/go/elrond-go/cmd/node/logs
-     - ~/MyObservingSquad/node-0/config:/config
+     - ${SQUAD_DIRECTORY}/node-0/db:/go/mx-chain-go/cmd/node/db
+     - ${SQUAD_DIRECTORY}/node-0/logs:/go/mx-chain-go/cmd/node/logs
+     - ${SQUAD_DIRECTORY}/node-0/config:/config
    command: --destination-shard-as-observer=${SHARD_0} --validator-key-pem-file=/config/observerKey_${SHARD_0}.pem --display-name="${DISPLAY_NAME_0}"
    networks:
-     elrond-squad:
+     multiversx-squad:
        ipv4_address: 10.0.0.6
 
  observer-1:
-   image: elrondnetwork/${NODE_TAG}
+   image: ${ORGANIZATION}/${OBSERVER_IMAGE}
    ports:
      - "8081:8080"
      - "10001:37373"
    environment:
      - SHARD=1 
    volumes:
-     - ~/MyObservingSquad/node-1/db:/go/elrond-go/cmd/node/db
-     - ~/MyObservingSquad/node-1/logs:/go/elrond-go/cmd/node/logs
-     - ~/MyObservingSquad/node-1/config:/config
+     - ${SQUAD_DIRECTORY}/node-1/db:/go/mx-chain-go/cmd/node/db
```

Comment:
might document in the Readme or Docs how to rename volumes for a smooth transition to the new observing squad version(s)

Reply:
I will do some tests to verify if the upgrade with the new squad works correctly with the previous version.